parley/src/tests/mod.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright 2024 the Parley Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+// TODO - Figure out which tests can run without std
```

Comment:
Hmm, why do we want this? Afaik, we only enable `no_std` when not running tests.

Reply:
IIRC, the test runner itself requires std so tests are always run with std regardless of whether you enable it anyway.